data/releases/schedule.yaml

```diff
@@ -44,13 +50,16 @@ schedules:
     - release: 1.20.1
       cherryPickDeadline: 2020-12-11
       targetDate: 2020-12-18
-      note: "Tagging Issue https://groups.google.com/g/kubernetes-dev/c/dNH2yknlCBA"
+      # note: "Tagging Issue https://groups.google.com/g/kubernetes-dev/c/dNH2yknlCBA"
```

Comment:
What's the reason to comment that?

Reply:
no reason, push by mistake while fixing the schedule-builder

Reply:
fixed, thanks for the review @saschagrunert